aospy/model.py

```diff
@@ -89,15 +89,38 @@ def _get_grid_attr(self, grid_objs, attr_name):
                 pass
         return
 
+    def _rename_coords(self, ds):
+        """Renames all coordinates within a Dataset or DataArray so that they match 
+        the internal names.
+        """
+        primary_attrs = {
+            'lat':         ('lat', 'latitude', 'LATITUDE', 'y', 'yto'),
```

Comment:
Let's use the package constants for the keys here, e.g. LAT_STR, LON_STR, etc.  (And anywhere else)